Apps/Src/ReadCarCAN.c

```diff
@@ -2,40 +2,7 @@
 
 #include "ReadCarCAN.h"
 
-#define THRESHOLD 3
 
 void Task_ReadCarCAN(void *p_arg) {
```

Comment:
We need to merge master into this branch, or else this will overwrite the most recent SendCarCAN merge.

Reply:
Ok